packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -78,14 +78,14 @@ export default class AgoraIpOrderCtrl {
       .all({
         user: this.User.getUser(),
         services: this.IpAgoraOrder.getServices(),
-        ipFailoverRIPEPrice: this.IpAgoraOrder.getIpFailoverRIPEPrice(
+        ipFailoverPrice: this.IpAgoraOrder.getIpFailoverPrice(
```

Comment:
**nitpick** : maybe manage it with default value will be safer no ?

Reply:
Seen together we already have a default value for region.